Update CONTRIBUTING.md

chawinccmkforlearn opened the pull request.

py3-coder commented:
Hey @chawinccmkforlearn ,
Need one update form you.
We have changed the guideline since many of contributor facing issue so we change it.
Just delete the contribution.md
Will accept the PR for sure

chawinccmkforlearn commented:
> Hey @chawinccmkforlearn , Need one update form you. We have changed the guideline since many of contributor facing issue so we change it. Just delete the contribution.md Will accept the PR for sure

deleted the contribution.md

py3-coder commented:
PR accepted @chawinccmkforlearn 
Successfully merged to HacktoberFest_22 repo.
🚀🚀👍
